app/controllers/verify/finance_controller.rb

```diff
@@ -31,7 +31,7 @@ def step_name
     end
 
     def confirm_step_needed
-      redirect_to verify_address_path if idv_session.financials_confirmation == true
+      redirect_to redirect_url if idv_session.financials_confirmation == true
```

Comment:
Maybe I'm misunderstanding, but I don't think the `verify/address` page should be restricted since it allows the user to choose between USPS and Phone verification. If the USPS feature is ON, then the USPS button should show up on the `verify/address` page, otherwise, it should not be shown. In other words, unless this controller was doing something wrong, it should not be changed. What should be changed is the logic in the view to determine when to show the USPS button since address verification can be performed in one of two ways currently. It's the USPS verification that needs to be toggled on or off, not address verification as a whole, right?